### PR TITLE
Developed UI for Quiz Intake Form (WIP)

### PR DESCRIPTION
**Pull request by jimip85**
Hey Lucky, 

In this PR, I developed: 
- UI for the first shared component
- intake logic for onboarding i.e. handleChange
- the handleSubmit function

**Figma Design:**
image

**WIP UI:**
image
image (2)

### CODE REVIEW

File: pages/tech-per-quiz.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        const { name, value } = e.target
+        setOnBoarding((prev) => {
+            return { ...prev, [name]: value };
+        });
+    };
+
+    const handleSubmit = (e) => {
+        // e.preventDefault();
+        console.log(onBoarding);
+    };
```
Comment (LuckyHariharan) [inline pages/tech-per-quiz.tsx:20]: Great work, do you mind moving the handle functions to their own component, keep this in the same folder

**Review by LuckyHariharan — COMMENTED** [review]
Overall great work! Will ping Vithushan for review.

File: pages/studentPortal/quizcomponent/tech-per-quiz.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    const handleChange = (e) => {
+        const { name, value } = e.target
+        setOnBoarding((prev) => {
+            return { ...prev, [name]: value };
+        });
+    };
+
+    const handleSubmit = (e) => {
+        // e.preventDefault();
```
Comment (vithushan19) [inline pages/studentPortal/quizcomponent/tech-per-quiz.tsx:18]: it's bad practice to commit commented code. better to remove this function entirely for now, if it's not needed at the moment. you can create a follow up pr to wire up the submit button.

**Review by vithushan19 — APPROVED** [review]
looks good to me

**Pull request merged**